src/v/raft/replicate_batcher.cc

```diff
@@ -145,8 +145,6 @@ ss::future<> replicate_batcher::flush(ss::semaphore_units<> u) {
              item_cache = std::move(item_cache),
              batcher_units = std::move(batcher_units)](
               ss::semaphore_units<> u) mutable {
-                // release batcher replicate batcher lock
-                batcher_units.return_all();
```

Comment:
```
to prevent append entries requests from being reordered.
```

I was under the impression that this PR was about applying back pressure, and the other PR was about dealing with re-ordering.

Reply:
you are right, this is not required, I will drop this commit. Sorry.